distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -38,9 +38,7 @@ type Table
        - connection: The connection with which the table is associated.
        - internal_columns: The internal representation of the table columns.
        - context: The context associated with this table.
-    # type Table (name : Text) (connection : Connection)
-    #            (internal_columns : Vector Internal_Column)
-    #            (context : IR.Context)
+    # type Table (name : Text) (connection : Connection) (internal_columns : Vector Internal_Column) (context : IR.Context)
     Table_Data name connection internal_columns context
```

Comment:
This comment is out of date vs the constructor, we should update it to say Table_Data.
But more importantly - the comment was added because the type annotation syntax used to crash the parser. I don't think this is the case, so if you are already modifying these declarations, maybe we can just remove the comments and put types where they belong?

```suggestion
    Table_Data (name : Text) (connection : Connection) (internal_columns : Vector Internal_Column) (context : IR.Context)
```

Reply:
(and we can do the same for the other types, it would be good to get rid of these legacy comments - as long as the parser supports this correctly now, but I assume it does)

Reply:
I have fixed some but reviewing all the type signatures is a larger job and one we need to do - don't want that to be part of this PR but will fix up the ones touched by the PR.